src/book/components/BookDetails/BookDetails.tsx

```diff
@@ -44,8 +44,16 @@ export const BookDetails = () => {
     }
   }, [id]);
 
-  const notifyOnBookChange = (data: BookProperties) => {
-    save({ id: +id!, ...data }).then(() => navigate("/book-app/books"));
+  const navigateToBookList = () => navigate("/book-app/books");
+
```

Comment:
saveNew was not working here, needed to be fixed, need to incorporate this to previous modules if missing